A/armadillo/build_tarballs.jl

```diff
@@ -33,15 +33,15 @@ if [[ "${nbits}" == 64 ]]; then
     if [[ "${target}" == *-apple-* ]] || [[ "${target}" == *-mingw* ]]; then
         FLAGS+=(-DALLOW_OPENBLAS_MACOS=ON)
     fi
-    if [[ "${target}" == powerpc64le-* ]]; then
-        FLAGS+=(-DCMAKE_CXX_FLAGS="-fopenmp")
-    fi
 
     for sym in cgbcon cgbsv cgbsvx cgbtrf cgbtrs cgecon cgees cgeev cgeevx cgehrd cgels cgelsd cgemm cgemv cgeqp3 cgeqrf cgesdd cgesv cgesvd cgesvx cgetrf cgetri cgetrs cgges cggev cgtsv cgtsvx cheev cheevd cherk clangb clange clanhe clansy cpbtrf cpocon cposv cposvx cpotrf cpotri cpotrs cpstrf ctrcon ctrsyl ctrtri ctrtrs cungqr dasum ddot dgbcon dgbsv dgbsvx dgbtrf dgbtrs dgecon dgees dgeev dgeevx dgehrd dgels dgelsd dgemm dgemv dgeqp3 dgeqrf dgesdd dgesv dgesvd dgesvx dgetrf dgetri dgetrs dgges dggev dgtsv dgtsvx dlahqr dlangb dlange dlansy dlarnv dnrm2 dorgqr dpbtrf dpocon dposv dposvx dpotrf dpotri dpotrs dpstrf dstedc dsyev dsyevd dsyrk dtrcon dtrevc dtrsyl dtrtri dtrtrs ilaenv sasum sdot sgbcon sgbsv sgbsvx sgbtrf sgbtrs sgecon sgees sgeev sgeevx sgehrd sgels sgelsd sgemm sgemv sgeqrf sgeqp3 sgesdd sgesv sgesvd sgesvx sgetrf sgetri sgetrs sgges sggev sgtsv sgtsvx slahqr slangb slange slansy slarnv snrm2 sorgqr spbtrf spocon sposv sposvx spotrf spotri spotrs spstrf sstedc ssyev ssyevd ssyrk strcon strevc strsyl strtri strtrs zgbcon zgbsv zgbsvx zgbtrf zgbtrs zgecon zgees zgeev zgeevx zgehrd zgels zgelsd zgemm zgemv zgeqp3 zgeqrf zgesdd zgesv zgesvd zgesvx zgetrf zgetri zgetrs zgges zggev zgtsv zgtsvx zheev zheevd zherk zlangb zlange zlanhe zlansy zpbtrf zpocon zposv zposvx zpotrf zpotri zpotrs zpstrf ztrcon ztrsyl ztrtri ztrtrs zungqr; do
         SYMB_DEFS+=("-D${sym}=${sym}_64")
     done
 
     export CXXFLAGS="${SYMB_DEFS[@]}"
+    if [[ "${target}" == powerpc64le-* ]]; then
+        CXXFLAGS+=(-fopenmp)
+    fi
```

Comment:
I think you need to put this before the `export`, which flattens the array into a string.  I'm not 100% sure on that, though.

Reply:
Sorry, I think my suggestion was bad... maybe it suffices to just do this:

```
if [[ "${target}" == powerpc64le-* ]]; then
    export CXXFLAGS="${SYMB_DEFS[@]} -fopenmp"
else
    export CXXFLAGS="${SYMB_DEFS[@]}"
fi
```

(You may want to double-check that, I didn't test it or anything.  Maybe printing `$CXXFLAGS` to ensure it's correct is worthwhile.)